AWS-CodePipeline/sam-monorepo-pipeline-template/{{cookiecutter.outputDir}}/pipeline/buildspec_build_package.yml

```diff
@@ -0,0 +1,29 @@
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      python: 3.9
+    commands:
+      - pip install --upgrade pip
+      - pip install --upgrade awscli aws-sam-cli
+      # Enable docker https://docs.aws.amazon.com/codebuild/latest/userguide/sample-docker-custom-image.html
+      - nohup /usr/local/bin/dockerd --host=unix:///var/run/docker.sock --host=tcp://127.0.0.1:2375 --storage-driver=overlay2 &
+      - timeout 15 sh -c "until docker info; do echo .; sleep 1; done"
+  build:
+    commands:
+      - cd ${PROJECT_SUBFOLDER}
```

Comment:
Assuming this is passed as-is to a shell, this (and all other unquoted usage) will fail if the value contains spaces. Should be `"${FOO}"`.

Reply:
Very good point, thanks for that! Fixed!